### PR TITLE
Remove util type checks

### PR DESCRIPTION
**Pull request by guilhermemntt**
Closes https://github.com/decred/decrediton/issues/3131.

### CODE REVIEW

File: app/actions/StatisticsActions.js [hunk 1]
```diff
@@ -391,7 +390,7 @@ export const exportStatToCSV = (opts) => (dispatch, getState) => {
   // called once for each data line
   const progressFunction = (time, series) => {
     const values = allSeries.map((s) =>
-      !isUndefined(series[s.name]) ? s.valueFormatFunc(series[s.name]) : null
+      series[s.name] !== undefined ? s.valueFormatFunc(series[s.name]) : null
```
Comment (amassarwi) [inline app/actions/StatisticsActions.js:393]: This isn't enough, you could also check if typeof x === "undefined" but I would rather user lodash `isUndefined` func for that.

Lodash may have it's own version of `isNumber, isNullOrUndefined` funcs as well, checkout their documentation.

**Review by amassarwi — CHANGES_REQUESTED** [review]
Thanks for this PR, left an inline suggestion.

**Comment by guilhermemntt** [comment]
Thanks @amassarwi, you're totally right. Just to be clearer: ``something === "undefined"`` throws error if ``something`` hasn't been defined yet. So, there was a trap in ``@deprecated`` annotation and its suggestion. Following your suggestion now, ``lodash`` has all these type check, including [isNil](https://lodash.com/docs/4.17.15) which is equivalent to ``isNullOrUndefined``.

**Review by amassarwi — APPROVED** [review]
Noice 👍

**Comment by vctt94** [comment]
needs rebase

**Comment by guilhermemntt** [comment]
Rebased.

**Comment by alexlyp** [comment]
Quite a few conflicts that will need to be dealt with.

**Review by matheusd — APPROVED** [review]
Nice job. Thanks for this!

**Pull request merged**